entity-service/src/integrationTest/resources/configs/entity-service/application.conf

```diff
@@ -11,5 +11,20 @@ entity.service.config = {
     }
   }
 }
-# This should be completely driven based on config given in app packaging.
-entity.service.attributeMap = []
+entity.service.attributeMap = [
+  {
+    "scope": "SERVICE",
+    "name": "SERVICE.id",
+    "subDocPath": "entityId"
+  },
+  {
+    "scope": "SERVICE",
+    "name": "SERVICE.name",
+    "subDocPath": "entityName"
+  },
+  {
```

Comment:
you want to add one more attribute with `attributes.` as it has a special condition in code.
e.g `attributes.API_NAME`

Reply:
Its being tested in testEntityQueryServiceImplTest#testConvertToEntityQueryResult